File: Python_version/angiogenesis_bm.py
from functions.brownian_motion import brownian_motion as bm
import functions.grad_div as gd
import functions.phi_calc as p
import parameters
import numpy as np
# definition of parameters is now in parameters.py

# initialization of arrays

nreps = int(1e0)

Xdata = dict() # this dictionary is going to store the data of each rep for the coordinate
Da_data = dict() # this dictionary is going to store the data of each rep for the diffusion gradient

for ix in range(nreps):
    Xem = np.zeros((parameters.N,2)) # array to store the coordinates
    Vem = np.zeros((parameters.N,2)) # array to store the velocity
    Xzero = np.array([0,0]) # initial condition coordinate
    Vzero = np.array([0,0]) # initial condition velocity
    Xem[0,:] = Xzero
    Vem[0,:] = Vzero
    # first angles
    theta = np.pi/2
    phi = np.pi/2
    theta_list = []
    Da_list = []

    # initialization of the loop for generate the motion of the sprout
    dW, W = bm(parameters.N, parameters.T, dim = 2)
    for j in range(0,parameters.L):
        print(j)
        

File: Julia_version/angiogenesis_bm.jl
include(joinpath("functions","BrowniaMotion.jl"))
using .BrownianMotion
nReps = 1e0

Xdata = Dict()
Da_data = Dict()

T = 3
N = 2000
dt = T/N

# parameters of the analysis
beta = 1/3 # h^-1
alpha = 40 * 1/3 # µm^2 h^-3
kappa = 10 # arbitrary parameter

for ix in 1:nReps

    

    # generate the fractional Brownian motion   

    bm = BrownianMotion.(N,T)

    Xzero = [0,0]
    Xtemp = Xzero

    

end
